base/applications/regedit/lang/pt-PT.rc

```diff
@@ -149,7 +149,7 @@ BEGIN
         MENUITEM "Modificar dados binários", ID_EDIT_MODIFY_BIN
         MENUITEM SEPARATOR
         MENUITEM "&Excluir\tDel", ID_EDIT_DELETE
-        MENUITEM "&Mudar o nome", ID_EDIT_RENAME
+        MENUITEM "&Renomear", ID_EDIT_RENAME
```

Comment:
Where do these translation changes come from?

Reply:
From an older branch. Intention is to avoid accelerator collision, see commit comment.

Reply:
So does that imply that a language maintainer has since updated these translations to be more correct/natural/consistent and you're reverting their decision? Or why are the translations different in the older branch?

Reply:
If you prefer me to keep the current translation and just change the accelerator position within the string, just let me know. That would be possible as well, but results in longer strings. I had the impression that the older translators invested more love into the proper translations thn their younger colleagues. Therefore I relied on "the ancient wisdom of the elders" in some cases.

Reply:
Obviously you should keep the current translation from current master, and just fix the keyboard accelerators. You are in no position to judge whether the old translations were better or whatever. If they have changed in master, this proves that the old ones were not that great.

Reply:
The new ones just prove that the translator did not know a thing about what accelerators were and how to use them without causing duplicates and therefore breaking 2 accelerators from working at the same time with his changes. They are definitely not "better", otherwise they would work in master head. And they don't, due to the collisions.

Reply:
Text translation and keyboard accelerators are two different things, even though they meet each other at the same place. The translator improved the text, but indeed at the detriment of the accelerators.

Reply:
I still don't understand why you don't just move the & to the correct letter then? "Muda&r o nome" and all is set. I hope your reply is not "Because the older translation is 4 Bytes shorter"... If that is the case I feel the urge to review any and every german translation for shortening out of no reason.